stores/amazon_requests.py

```diff
@@ -499,15 +521,12 @@ def run(self, delay=5, test=False, all_cookies=False):
         # Transfer cookies from selenium session.
         # Do not transfer cookies to stock check if using proxies
         if not self.proxies:
-            transfer_selenium_cookies(
-                self.driver, self.session_stock_check, all_cookies=all_cookies
```

Comment:
I actually need to test ditching the cookie transfer for stock checking completely.

Reply:
Nevermind, I've commented this out, so same effect.